src/geometamaker/geometamaker.py

```diff
@@ -210,13 +214,12 @@ def __init__(self, source_dataset_path=None):
                 self.mcf = _get_template(MCF_SCHEMA)
                 self.mcf['metadata']['identifier'] = str(uuid.uuid4())
 
-                # fill all values that can be derived from the dataset
-                self._set_spatial_info()
-                self.mcf['metadata']['datestamp'] = datetime.utcnow(
-                    ).strftime('%Y-%m-%d')
+            # fill all values that can be derived from the dataset
+            self._set_spatial_info()
```

Comment:
Making sure to call this whether or not an MCF already exists, in case properties of the dataset have changed.